comdb2/__init__.py

```diff
@@ -28,4 +28,4 @@
 connections, this module may be simpler to get started with.
 """
 
-__version__ = "1.1.0"
+__version__ = "1.1.6"
```

Comment:
Should this be `1.1.1`?  What's the motivation for skipping a few patch versions?

Reply:
We've done a poor job of keeping `__version__` updated.  The current version is actually 1.1.5, this is catching `__version__` up (and preparing for a new release by adding 1).

Reply:
Rodger.